src/NzbDrone.Core/Notifications/Boxcar/BoxcarProxy.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using FluentValidation.Results;
+using NLog;
+using RestSharp;
+using NzbDrone.Core.Rest;
+
+namespace NzbDrone.Core.Notifications.Boxcar
+{
+    public interface IBoxcarProxy
+    {
+        void SendNotification(string title, string message, BoxcarSettings settings);
+        ValidationFailure Test(BoxcarSettings settings);
+    }
+
+    public class BoxcarProxy : IBoxcarProxy
+    {
+        private readonly Logger _logger;
+        private const string URL = "https://new.boxcar.io/api/notifications";
```

Comment:
The URL is in 404 with a GET request.
You have to do a POST on this url with parameters :
https://new.boxcar.io/api/notifications?user_credentials=accessToken&notification[title]=testTitle&notification[long_message]=testMessage